src/main/resources/db/changesets/20200717_allowlongsnapshotrelnames.yaml

```diff
@@ -0,0 +1,9 @@
+databaseChangeLog:
+  - changeSet:
+      id: allowlongsnapshotrelnames
+      author: danmoran
+      changes:
+        - modifyDataType:
+            tableName: snapshot_relationship
+            columnName: name
+            newDataType: text
```

Comment:
If we change this to something length-limited, the migration will need to be extended to also impose the limit on the dataset_relationship table.